azure/new-workspace/README.md

```diff
@@ -1,10 +1,75 @@
+| **client_secret**                | Secret of the Terraform App Registration                           | Yes                   | String       |                     | XXXXXXXxxxxxYYYYYYyyyyyyyZZZZZZzzzzz  |
+| **app_platform_name**            | Name of the Platform App Registration                              | Yes                   | String       |                     | Cosmo Tech Platform for Project       |
+| **app_adt_name**                 | Name of the Network ADT App Registration                           | Yes                   | String       |                     | Cosmo Tech Network ADT for Project    |
+| **dedicated_eventhub_namespace** | Create a dedicated eventhub namespace (true if multiple workspaces on the Platform)| Yes   | String       | true                | false                                 |
+| **eventhub_namespace_name**      | Name of the Event Hub Namespace deployed by default on the platform| Yes if dedicated_eventhub_namespace=false | String       | | evnamespacexxxxx                      |
+| **adx_name**                     | Name of the Azure Data Explorer resource deployed by the Platform  | Yes                   | String       |                     | kustoxxxxxxx                          |
+| **storage_account_name**         | Name of the Azure Storage Account resource deployed by the Platform| Yes                   | String       |                     | storagexxxxxxx                        |
+| **subscription_id**              | ID of the subscription where the Cosmo Tech Platform is deployed   | Yes                   | String       |                     | abcdefgh-1234-5678-abcd-123456789ABC  |
+| **resource_group**               | Name of the managed resource group of the Cosmo Tech Platform      | Yes                   | String       |                     | mrg-cosmotechsdtplatform-20230101     |
+| **organization_id**              | ID of the Organization previously created in the API               | Yes                   | String       |                     | o-xxxxxxxxxx                          |
+| **workspace_key**                | Key of the workspace previouly created in the API                  | Yes                   | String       |                     | supplychainworkspace                  |
+| **eventhub_namespace_capacity**  | Event Hub Namespace capacity in terms of throughput units          | No                    | Integer      | 2                   | 2                                     |
+| **kusto_script**                 | Run the database script (recommended value is `true`)              | No                    | String       | true                | true                                  |
+| **aad_groups_and_assignements**  | Run the database script (recommended value is `false`)             | No                    | String       | false                | false                                 |
```

Comment:
typo ?

Reply:
where ? sorry I don't understand

Reply:
my bad
Here: `| **aad_groups_and_assignements**  | Run the database script (recommended value is `false`) `
I thought you probably wanted to say: "Create the Azure Active Directory workspace group and do roles assignements on resources" instead

Reply:
you are right, it is a mistake :)

Reply:
@enocquet  I just fixed the issue.
Thanks